extensions/data-plane/data-plane-google-bigquery/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
the new extension needs to be referenced in the `settings.gradle.kts` file, otherwise it won't be built/published.
An E2E test would have helped you in discover that, please add at least one.

Reply:
I've updated the settings file, looking for a viable e2e test, possibly running in local